.github/CODEOWNERS

```diff
@@ -420,8 +420,14 @@
 
 # QA team.
 /core/tests/ @nithusha21
-/core/tests/protractor_utils @U8NWXD
-/core/tests/protractor_desktop @U8NWXD
+/core/tests/protractor_utils/ @U8NWXD
+/core/tests/protractor_desktop/ @U8NWXD
+/core/tests/protractor/ @U8NWXD
+/extensions/interactions/*/protractor.js @U8NWXD
+/extensions/interactions/protractor.js @U8NWXD
+/extensions/rich_text_components/*/protractor.js @U8NWXD
+/extensions/rich_text_components/protractor.js @U8NWXD
+/extensions/objects/protractor.js @U8NWXD
```

Comment:
@U8NWXD I think you can own `protractor*.js` file, which will include config file and any protractor.js file. What do you think?

Reply:
@DubeySandeep I don't think the `protractor*.js` pattern will work since `*` doesn't match directory separators (see the [gitignore syntax link](https://git-scm.com/docs/gitignore#_pattern_format) on the [GitHub codeowners docs](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-syntax))

Reply:
`protractor*.js`  will math anyfile which starts with `protractor` and endwith `.js`. This will include all protractor.js file as well as protractor.config.js file. Can you please check this? (not sure!)

Other option:
Use both: 
```
protractor.js @....
protractor.config.js @... (Also, add devflow team lead here maybe?)
```

Reply:
Though I'm fine with what you have right now!

Reply:
done!

Reply:
@DubeySandeep Using `protractor*.js` resulted in this linter error:

```
Pattern on line 424 is invalid. Use full path relative to the root directory
```

I am reverting to what I had before